app/views/billing_expenses/show.html.slim

```diff
@@ -1,30 +1,81 @@
 .row
-  .col-xs-12
-    .table-responsive
-      table.table.table-no-border-top
-        tbody
-          tr
-            td= t_model(Volunteer)
-            td= link_to @volunteer.contact.full_name, volunteer_path(@volunteer)
-          tr
-            td= t_attr(:address)
-            td= @volunteer.contact.full_address
-          tr
-            td= t_attr(:bank_details, Volunteer)
-            td= @billing_expense.full_bank_details
-          tr
-            td= t_attr(:waive, Volunteer)
-            td= t(@volunteer.waive)
-          tr
-            td= t_attr(:amount)
-            td= @billing_expense.amount
-          tr
-            td= t_attr(:created_at)
-            td= l(@billing_expense.created_at.to_date)
-          tr
-            td= t_attr(:created_by)
-            td= @billing_expense.user.profile.contact.full_name
+  br=
+  br=
+  .col-xs-11.col-xs-offset-1
+    h3
+      strong
+        = "#{t('billing_expense_pdf.title')} "
+        = "#{@billing_expense.volunteer.contact.full_name} "
+        - if @billing_expense.volunteer.hours.any?
```

Comment:
That looks if the volunteer for whom the billing expense has any hours. Even billed in other billings. 
What needed, is to check if there are any hours to be billed for this billing expense. 
The query of the method (let's name it billed_hours) that need to be implemented is
`Hour.where(billing_expense: @billing_expense)`
and then you will do:
`if @billing_expense.billed_hours.any?`
and then 
`@billing_expense.billed_hours.last.meeting_date`